restaurant-api/src/main/resources/application.properties

```diff
@@ -4,5 +4,6 @@ spring.datasource.url=jdbc:postgresql://localhost:5432/postgres
 spring.datasource.username=postgres
 spring.datasource.password=password
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
-spring.jpa.hibernate.ddl-auto=create
+spring.jpa.hibernate.ddl-auto=create-drop
```

Comment:
We'll want to change this to update when all the services are together, can be done in external configuration so don't worry to much though.